prac_08/taxi_simulator_program.py

```diff
@@ -0,0 +1,40 @@
+from prac_08.silver_service_taxi import SilverServiceTaxi
+from prac_08.taxi import Taxi
+
+
+taxis = [Taxi(100, "Prius"), SilverServiceTaxi(100, "Limo", 2),
+         SilverServiceTaxi(200, "Hummer", 4)]
+
+total_cost = 0
+taxi_selection = ""
```

Comment:
This should not be a string. When will it ever be a valid string? It's meant to store an object, so use `None`.